.env.example

```diff
@@ -90,6 +90,9 @@ DAYS_TO_EXPIRY_2=
 DAYS_POST_EXPIRY=
```

Comment:
@blabadi if you have thoughts on better env var/config val names i'm all ears...these feel sub-par